charts/seed-bootstrap/templates/gardener-resource-manager/deployment.yaml

```diff
@@ -0,0 +1,41 @@
+---
+apiVersion: {{ include "deploymentversion" . }}
+kind: Deployment
+metadata:
+  name: gardener-resource-manager
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: gardener-resource-manager
+spec:
+  revisionHistoryLimit: 0
+  replicas: {{ .Values.gardenerResourceManager.replicas }}
+  selector:
+    matchLabels:
+      app: gardener-resource-manager
+  template:
+    metadata:
+      {{- if .Values.gardenerResourceManager.podAnnotations }}
+      annotations:
+{{ toYaml .Values.gardenerResourceManager.podAnnotations | indent 8 }}
+{{- end }}
+      labels:
+        app: gardener-resource-manager
+        networking.gardener.cloud/to-dns: allowed
```

Comment:
These networking labels are not needed, I guess. We only install network policies to the shoot namespaces.